Add CLI Examples for CloudWatch Internet Monitor (#9005)
aws/aws-cli: awscli/examples/internetmonitor/create-monitor.rst

**To create a new internet monitor**

> This example uses the ``create-monitor`` command to create a new internet monitor. ::

Reviewer: ```suggestion
The following ``create-monitor`` example creates a new internet monitor. ::
```

All your examples need their intro lines updated to follow the format of "The following ``command-name`` example..."
https://aws.github.io/aws-cli/docs_styleguide.html#description

    aws internetmonitor create-monitor \
        --monitor-name TestMonitor \
        --resources arn:aws:ec2:us-east-1:123456789012:vpc/vpc-123456abcdef \
        --traffic-percentage-to-monitor 100 \
        --internet-measurements-log-delivery S3Config={LogDeliveryStatus=DISABLED} 

Output::

    {
        "Arn": "arn:aws:internetmonitor:us-east-1:123456789012:monitor/TestMonitor",
        "Status": "PENDING"

>     }

Reviewer: All your examples need a "For more information..." line with a link.
https://aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link
